python/py-uritools/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-uritools
+version             4.0.2
+revision            0
+
+supported_archs     noarch
+license             MIT
+maintainers         {@catap korins.ky:kirill} openmaintainer
+
+description         URI parsing, classification and composition
+long_description    {*}${description}
+
+homepage            https://github.com/tkem/uritools/
+
+checksums           rmd160  a55def1e60764c671a1cea7c3fd87ebaee68dac0 \
+                    sha256  04df2b787d0eb76200e8319382a03562fbfe4741fd66c15506b08d3b8211d573 \
+                    size    23673
+
+python.versions     310 311 312
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
```

Comment:
dependency is implied as it is using "python.pep517 yes" with the default "setuptools"